circle.yml

```diff
@@ -0,0 +1,19 @@
+machine:
+  services:
+    - docker
+
+test:
```

Comment:
`test` goes after `database`.  You want to try and keep these in the order they happen in circle. https://circleci.com/docs/manually/#overview